From bb2f90c07f0356dfc853122b94863c971a9f14b1 Mon Sep 17 00:00:00 2001
From: Sebastien Merle <sebastien.merle@stritzinger.com>
Date: Wed, 23 Jul 2025 14:51:55 +0000
Subject: [PATCH] arm64: dts: Disable broken imx8mm-kontron-bl-extension build

The imx8mm-kontron-bl-extension device tree has incorrect Makefile
configuration that expects .dtbo overlay source but the file is
provided as .dts.

This causes build failure:
"No rule to make target 'imx8mm-kontron-bl-extension.dtbo'"

Since this device tree is not needed for basic BL hardware operation
(it adds camera and PCIe extension board support), disable it to
allow the build to proceed.

Users who need extension board functionality can manually enable
this after fixing the overlay configuration.
---
 arch/arm64/boot/dts/freescale/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index ef124e410044..7c9948d43459 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -263,11 +263,11 @@ dtb-$(CONFIG_ARCH_MXC) += imx93-tqma9352-mba93xxla.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx93-var-som-symphony.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx95-19x19-evk.dtb

-imx8mm-kontron-bl-extension-dtbs	:= imx8mm-kontron-bl.dtb imx8mm-kontron-bl-extension.dtbo
+#imx8mm-kontron-bl-extension-dtbs	:= imx8mm-kontron-bl.dtb imx8mm-kontron-bl-extension.dtbo
 imx8mm-kontron-dl-dtbs			:= imx8mm-kontron-bl.dtb imx8mm-kontron-dl.dtbo
 imx8mm-kontron-bl-lte-dtbs		:= imx8mm-kontron-bl.dtb imx8mm-kontron-bl-lte.dtbo

-dtb-$(CONFIG_ARCH_MXC) += imx8mm-kontron-bl-extension.dtb
+#dtb-$(CONFIG_ARCH_MXC) += imx8mm-kontron-bl-extension.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-kontron-dl.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-kontron-bl-lte.dtb

--
2.43.0
